OpenShift-Deployment-Guide.md

```diff
@@ -0,0 +1,54 @@
+I was so happy. Took me more than 10 days to finish my voting app, my first web app project at Free Code Camp. I was ready to deploy to heroku and...
+
+![](http://i.imgur.com/fY2be3a.jpg)
+
+After some search at Google I found OpenShift. Then I went to Quincy Larson to know if was any problem to deploy to OpenShift instead of Heroku, he said it was Ok and ask me if I could document the process, and here it is...
+
+# Requirements
+- An account in [OpenShift](https://www.openshift.com/app/account/new)
+- Our app in a Git Repository
+
+###### Changes in your code
+- ```app.listen``` with ```process.env.OPENSHIFT_NODEJS_PORT``` and ```process.env.OPENSHIFT_NODEJS_IP```, both requires.
```

Comment:
No need to use three backticks in this instance, just one on each side will do the trick!